public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -10,6 +10,7 @@ import { CommandPalette } from 'app/features/commandPalette/CommandPalette';
 import { KioskMode } from 'app/types';
 
 import { AppChromeMenu } from './AppChromeMenu';
+import { MegaMenu as DockedMegaMenu } from './DockedMegaMenu/MegaMenu';
```

Comment:
basically because as soon as this toggle is live, i expect to remove the whole `MegaMenu` and will rename the `DockedMegaMenu` folder to just be `MegaMenu`. hope that makes sense 🙏

Reply:
Absolutely, thanks for the explanation 👍